documentation/how-to/optimize-deployment-performance.md

```diff
@@ -0,0 +1,42 @@
+# Optimise your deployment performance
+This guide describes steps you can take to optimize the performance of your superset deployment.
+
+## Enable asynchronous querying
+To enable asynchronous querying, you need to deploy Charmed Superset Workers. These workers handle long-running queries asynchronously, allowing the User Interface (UI) to remain responsive. You can do this as follows:
+```bash
+juju deploy superset-k8s --config charm-function=worker superset-k8s-worker
+```
+In order to be functional the Charmed Superset Worker requires relations with the same PostgreSQL and Redis charms as the server. These can be added as below:
+
+```bash
+juju relate superset-k8s-worker postgresql-k8s
+juju relate superset-k8s-worker redis-k8s
+```
+Using the UI, you can enable Asynchronous Query Execution (`AGE`) at the database level.
```

Comment:
question: Is it `AGE` or `AQE`? Also, we never refer to this again, perhaps there is no need for the acronym here.

Reply:
It is `AQE` you are correct, this was a typo. I think it is good to leave it as in the Superset UI it is often referred to just by this acronym, so it would be good the user of this guide can easily spot it.